src/components/NextButton.svelte

```diff
@@ -0,0 +1,9 @@
+<div class="row justify-center mt-8">
+    <button class="btn btn-primary my-auto" on:click={() => $swiper.slideNext() }>
+        <slot></slot>
+    </button>
+</div>
+
+<script>
+    import { swiper } from '../stores'
+</script>
```

Comment:
Please put the `script` at the top so you don't wonder where `$swiper` is coming from.

Reply:
✅ in 9e2b084